### PR TITLE
Add score board

### PR DESCRIPTION
**Pull request by choucl**
- Add score board to check scores for both players
- Maximize log length has been changed to 53

### CODE REVIEW

File: inc/render.h [hunk 1]
```diff
@@ -12,6 +12,8 @@ extern const shape_t tile_relation[4][6];
 #define GREEN_PAIR 3
 #define YELLOW_PAIR 4
 
+#define MAX_LOG_LEN 53
+
```
Comment (yuyuranium) [inline inc/render.h:16]: What a good practice.

File: src/render.c [hunk 2]
```diff
@@ -201,18 +201,50 @@ int render_message_log(char *msg[7], int *color[7])
...
+        } else if (gcb->score[i] < gcb->score[!i]) {
+            attron(COLOR_PAIR(RED_PAIR));
+        }
+        mvprintw(20 + i, 59, "Player %d: %2d", i, gcb->score[i]);
+        if (gcb->score[i] > gcb->score[!i]) {
+            attroff(COLOR_PAIR(GREEN_PAIR));
+        } else if (gcb->score[i] < gcb->score[!i]) {
+            attroff(COLOR_PAIR(RED_PAIR));
+        }
+    }
```
Comment (yuyuranium) [inline src/render.c:247]: At first, I don't know why the score are changing colors between green and red. After reading the code I finally realize the reason.

BTW, I usually use `p` for short when indicating the current player.

**Review by yuyuranium — APPROVED** [review]
I've checked out the branch and played around. It feels like a real game! What a great work!

File: src/render.c (hunk 2 above)
Comment (choucl) [reply src/render.c:247]: That sounds good, I'll change that.

**Pull request merged**